README.md

```diff
@@ -2089,6 +2092,22 @@ You may add additional, custom analytic events to templates using certain data a
 </button>
 ```
 
+## Built-In Analytics Events For CTAs
```

Comment:
Is this to be more explicit about the event types? https://github.com/yext/answers#calls-to-action
(They're listed as comments in the Calls to Action section)

Reply:
yes! especially for someone referencing this who is building things on jambo. I'm open to removing it in the Calls to Action section if you think it's redundant?

Reply:
ah nice yeah. can we remove and refer to this section in the Calls to Action section? is harder to remember to change in two places if this list ever gets updated